tests/e2e-desktop/scripts/wait-for-mount.ps1

```
@@ -0,0 +1,32 @@
+
+$ErrorActionPreference = "Stop"
+
+$Timeout = if ($env:MOUNT_TIMEOUT) { [int]$env:MOUNT_TIMEOUT } else { 90 }
+$Interval = 2
+$Elapsed = 0
+
+Write-Host "Waiting for mount at $MountPoint (timeout: ${Timeout}s)..."
+
+while ($Elapsed -lt $Timeout) {
+    if (Test-Path $MountPoint) {
+        Write-Host "PASS: Mount detected at $MountPoint"
```

PSScriptAnalyzer (1.24.0)

[warning] File 'wait-for-mount.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)